upf/upf.h

```diff
@@ -596,6 +597,7 @@ typedef struct
   urr_time_t time_quota;	/* relative duration in seconds */
   urr_time_t quota_holding_time;	/* relative duration in seconds */
   urr_abs_time_t monitoring_time;	/* absolute UTC ts since 1900-01-01 00:00:00 */
+  urr_time_t quota_validity_time;	/* relative duration in seconds */
```

Comment:
wrong order, Quota Validity Time should go before Monitoring Time to keep it in sync with TS 29.244.

Although it would also move before the urr_abs_time_t type just for readability.